index.html

```diff
@@ -767,21 +767,21 @@
 for the scale of processing implied by web systems), and experience with existing systems
 shows that they make it hard for [=people=] to exercise their rights.
 
-### Privacy Labour {#privacy-labour}
+### Privacy Labor {#privacy-labor}
 
-<dfn data-lt="privacy labor|labour|labor">Privacy labour</dfn> is the practice of having a [=person=] do
+<dfn data-lt="privacy labor|labor|labor">Privacy labor</dfn> is the practice of having a [=person=] do
```

Comment:
```suggestion
<dfn data-lt="labor">Privacy labor</dfn> is the practice of having a [=person=] do
```
Since we don't need to list the same spelling twice.